app/Navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Image from 'next/image';
+
+export default function Navbar() {
+	return (
+		<nav className="h-14 w-screen bg-white shadow-lg shadow-gray-400 sm:h-20 lg:mb-12 lg:h-24">
+			<div className="flex items-center justify-between sm:h-20 lg:h-24">
+				<div className="mb-2.5 ml-3 mt-2.5 sm:mb-3 sm:ml-20 sm:mt-4 lg:mb-4 lg:ml-36 lg:mt-6">
```

Comment:
instead of having the margins on the elements, why not just put padding sa parent?

you can do it like 
`<div className="flex items-center justify-between sm:h-20 lg:h-24 py-2.5 px-3 sm:py-3 sm:px-20 lg:px-36 lg:py-6">`